### ISSUE TITLE
[Suggestion] Skid the Elytrabot module from cookieclient

### ISSUE THREAD

**Issue by BookWyrm114** [AntiCope/meteor-rejects#97]
### Describe the feature

the Elytrabot from [Cookieclient](https://github.com/bebeli555/CookieClient) would be really useful for travel, and just a great feature overall.

**Comment by maxsupermanhd** [comment]
Reimplementing elytrabot would be amazing even with standalone module, very good feature

**Comment by SplashAni** [comment]
yeah bro

**Comment by maxsupermanhd** [comment]
https://github.com/RickyTheRacc/ElytraBruh
